src/main/webapp/index.html

```diff
@@ -19,9 +20,7 @@
         <input type="radio" name="option" id="actions-radio" value="actions" class="hide">
         <label for="actions-radio" id="actions-label" class="menu-label">Action</label>
       </div>
-      <div id="selected-option">
-        <div id="slogan"><p>Slogan</p></div>
```

Comment:
Why did you remove this div?

Reply:
Oh shoot, I meant to remove the forms, not the whole thing. Mb, I'll re-add it.